ci_framework/roles/edpm_deploy_baremetal/tasks/main.yml

```diff
@@ -44,14 +44,23 @@
   ansible.builtin.include_role:
     name: set_openstack_containers
 
-- name: Install Dataplane with baremetal
+- name: Create and provision virtual baremetal
```

Comment:
`provision` here is probably misleading.

Reply:
good point